app/styles/app.scss

```diff
@@ -113,6 +113,24 @@ ul.comma-list {
     color: black;
 }
 
+div[id='download_project clearfix'] {
```

Comment:
A few notes on CSS here:

1. You can reference an ID as `#name` which is slightly more idiomatic ([this post](http://stackoverflow.com/questions/9285451/handling-css-id-and-classes-with-spaces) claims the selector-based ID syntax is deprecated in newer browsers, but I can't find the evidence to support this claim)

2. Clearfix is usually a class name (from bootstrap)- it's weird to me that it would be part of the element ID, especially since...

3. ...support for element IDs with spaces is spotty at best, and generally should be avoided.

Out of curiosity, what happens if you rename this element so that clearfix is applied as a class to this parent element? Will the child styles still be necessary then?